packages/design-system/src/components/reset/resetAll.stories.tsx

```diff
@@ -19,7 +18,6 @@ export const Overview = () => {
       <Typography />
       <Lists />
       <FormElements />
-      <Tables />
```

Comment:
In removing redundant CSS, I found our table reset styles overreached into Table (they were identical). I removed the table reset story and styles as a result.